src/bank.js

```diff
@@ -1,3 +1,7 @@
+// Good job, this seems to work well. A couple of recommendations:
+// 1 - use a linter as spacing is off in quite a few places 
```

Comment:
you can set up linter by following the instructions here:

https://github.com/MCRcodes/eslint-config